R/api.R

```diff
@@ -33,17 +33,24 @@ root <- function() {
 
 ##' @porcelain 
 ##'   GET /report/list => json(report_list)
-##'   query hash :: string
+##'   query ref :: string
```

Comment:
I changed this to ref too because I think this is better description, it can work with a hash or a branch name